### PR TITLE
Param url

### PR DESCRIPTION
**Pull request by NethNick**
Added
- Open Parametric url on ring o after answer
- UI to choose Popup Call position

### CODE REVIEW

File: Nethifier/FRM_CONFIG.Designer.vb [hunk 1]
```diff
@@ -582,6 +597,126 @@ Partial Class FRM_CONFIG
...
+        Me.CHK_ADX.AutoSize = True
+        Me.CHK_ADX.CheckAlign = System.Drawing.ContentAlignment.MiddleRight
+        Me.CHK_ADX.ImageAlign = System.Drawing.ContentAlignment.MiddleRight
+        Me.CHK_ADX.Location = New System.Drawing.Point(287, 19)
+        Me.CHK_ADX.Name = "CHK_ADX"
+        Me.CHK_ADX.Size = New System.Drawing.Size(96, 17)
+        Me.CHK_ADX.TabIndex = 10
+        Me.CHK_ADX.Text = "In alto a destra"
+        Me.CHK_ADX.TextAlign = System.Drawing.ContentAlignment.MiddleRight
+        Me.TIP.SetToolTip(Me.CHK_ADX, "Definisce la posizione del PopUP che compare all'arrivo della chiamata")
```
Comment (gsanchietti) [inline Nethifier/FRM_CONFIG.Designer.vb]:
```suggestion
        Me.TIP.SetToolTip(Me.CHK_ADX, "Definisce la posizione del pop-up che compare all'arrivo della chiamata")

```

File: Nethifier/FRM_CONFIG.Designer.vb [hunk 2]
```diff
@@ -582,6 +597,126 @@ Partial Class FRM_CONFIG
...
+        Me.CHK_BDX.Checked = True
+        Me.CHK_BDX.CheckState = System.Windows.Forms.CheckState.Checked
+        Me.CHK_BDX.ImageAlign = System.Drawing.ContentAlignment.MiddleRight
+        Me.CHK_BDX.Location = New System.Drawing.Point(276, 42)
+        Me.CHK_BDX.Name = "CHK_BDX"
+        Me.CHK_BDX.Size = New System.Drawing.Size(107, 17)
+        Me.CHK_BDX.TabIndex = 9
+        Me.CHK_BDX.Text = "In basso a destra"
+        Me.CHK_BDX.TextAlign = System.Drawing.ContentAlignment.MiddleRight
+        Me.TIP.SetToolTip(Me.CHK_BDX, "Definisce la posizione del PopUP che compare all'arrivo della chiamata")
```
Comment (gsanchietti) [inline Nethifier/FRM_CONFIG.Designer.vb]:
```suggestion
        Me.TIP.SetToolTip(Me.CHK_BDX, "Definisce la posizione del pop-up che compare all'arrivo della chiamata")

```

File: Nethifier/FRM_CONFIG.Designer.vb [hunk 3]
```diff
@@ -582,6 +597,126 @@ Partial Class FRM_CONFIG
...
+        '
+        'CHK_BSX
+        '
+        Me.CHK_BSX.AutoSize = True
+        Me.CHK_BSX.Location = New System.Drawing.Point(6, 42)
+        Me.CHK_BSX.Name = "CHK_BSX"
+        Me.CHK_BSX.Size = New System.Drawing.Size(110, 17)
+        Me.CHK_BSX.TabIndex = 8
+        Me.CHK_BSX.Text = "In basso a sinistra"
+        Me.TIP.SetToolTip(Me.CHK_BSX, "Definisce la posizione del PopUP che compare all'arrivo della chiamata")
```
Comment (gsanchietti) [inline Nethifier/FRM_CONFIG.Designer.vb]:
```suggestion
        Me.TIP.SetToolTip(Me.CHK_BSX, "Definisce la posizione del pop-up che compare all'arrivo della chiamata")

```

File: Nethifier/FRM_CONFIG.Designer.vb [hunk 4]
```diff
@@ -582,6 +597,126 @@ Partial Class FRM_CONFIG
...
+        '
+        'CHK_ASX
+        '
+        Me.CHK_ASX.AutoSize = True
+        Me.CHK_ASX.Location = New System.Drawing.Point(6, 19)
+        Me.CHK_ASX.Name = "CHK_ASX"
+        Me.CHK_ASX.Size = New System.Drawing.Size(99, 17)
+        Me.CHK_ASX.TabIndex = 7
+        Me.CHK_ASX.Text = "In alto a sinistra"
+        Me.TIP.SetToolTip(Me.CHK_ASX, "Definisce la posizione del PopUP che compare all'arrivo della chiamata")
```
Comment (gsanchietti) [inline Nethifier/FRM_CONFIG.Designer.vb]:
```suggestion
        Me.TIP.SetToolTip(Me.CHK_ASX, "Definisce la posizione del pop-up che compare all'arrivo della chiamata")

```

File: Nethifier/FRM_CONFIG.Designer.vb [hunk 5]
```diff
@@ -582,6 +597,126 @@ Partial Class FRM_CONFIG
...
+        'CHK_ONEKEY
+        '
+        Me.CHK_ONEKEY.AutoSize = True
+        Me.CHK_ONEKEY.Enabled = False
+        Me.CHK_ONEKEY.Location = New System.Drawing.Point(53, 49)
+        Me.CHK_ONEKEY.Name = "CHK_ONEKEY"
+        Me.CHK_ONEKEY.Size = New System.Drawing.Size(308, 17)
+        Me.CHK_ONEKEY.TabIndex = 6
+        Me.CHK_ONEKEY.Text = "Usa un solo tasto per rispondere e per terminare la chiamata"
+        Me.TIP.SetToolTip(Me.CHK_ONEKEY, "Togliendo il check, la suoneria sarà esguita una volta sola all'arrivo della chia" &
```
Comment (gsanchietti) [inline Nethifier/FRM_CONFIG.Designer.vb]:
```suggestion
        Me.TIP.SetToolTip(Me.CHK_ONEKEY, "Disabilitando l'opzione, la suoneria sarà esguita una volta sola all'arrivo della chia" &

```

Or:
```suggestion
        Me.TIP.SetToolTip(Me.CHK_ONEKEY, "Togliendo la spunta, la suoneria sarà esguita una volta sola all'arrivo della chia" &

```

File: Nethifier/FRM_CONFIG.vb [hunk 6]
```diff
@@ -255,170 +272,177 @@ Friend Class FRM_CONFIG
...
+                        'Mylog("ResultCall", result.ToString())
+
+                        'Catch e As WebException
+                        'Mylog("ResultCall", e.Message.ToString())
+                        'MsgBox(e.Message.ToString(), 0, "ResultCall")
+                        'End Try
+                        '   'webClient.Dispose()
+                        'Else
+                        'Mylog("ResultCall", "Webclient Busy")
+                        'End If
```
Comment (gsanchietti) [inline Nethifier/FRM_CONFIG.vb]: Can we cleanup the unused code?

**Comment by gsanchietti** [comment]
Nethesis/dev#6141

**Comment by NethNick** [comment]
Fixed tooltips comments and removed unused code

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**